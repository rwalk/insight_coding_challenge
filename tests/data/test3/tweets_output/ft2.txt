0
4.5
2
2.5
2
1.0
2
1.5
1
1.0
1
1.0
1
1.0
1
1.0
1
1.5
2
2.5
3
3.5
4
3.5
4
3.5
3
3.5
3
3.5
3
3.0
3
2.5
2
2.5
3
2.5
3
3.0
3
3.0
3
3.0
3
3.5
3
3.5
4
3.5
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.0
4
4.5
5
5.0
5
5.0
5
5.0
5
5.0
5
5.0
