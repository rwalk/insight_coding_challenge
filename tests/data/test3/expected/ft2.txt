0
9
2
3
0
0
5
1
1
0
0
5
8
0
0
5
4
7
6
10
6
4
8
1
9
1
3
9
2
6
1
0
2
1
0
8
10
0
6
6
5
1
3
7
9
7
2
4
5
0
9
7
6
6
1
9
1
10
2
1
8
4
3
9
2
0
7
10
3
6
8
9
4
6
7
1
0
7
9
2
6
0
2
7
0
5
3
9
5
6
9
6
10
2
4
10
5
3
10
5
